src/components/DynamicFormContainer/DynamicForm/CustomTemplates/CustomObjectField.tsx

```diff
@@ -1,20 +1,19 @@
-import React, { ReactElement } from "react";
+import React, { FunctionComponent } from "react";
 import { ObjectFieldTemplateProps } from "react-jsonschema-form";
 
-export const CustomObjectFieldTemplate = ({
+export const CustomObjectFieldTemplate: FunctionComponent<ObjectFieldTemplateProps> = ({
   TitleField,
   properties,
   title,
   description,
   required,
-}: // eslint-disable-next-line  @typescript-eslint/no-explicit-any
-ObjectFieldTemplateProps<any>): ReactElement => {
+}) => {
   return (
     <>
       {title && <TitleField title={title} id={title} required={required} />}
       <ul className="dynamicForm-items">
-        {properties.map((prop) => (
-          <li className="my-4" key={prop.content.key ?? undefined}>
+        {properties.map((prop, index) => (
+          <li className="my-4" key={index}>
```

Comment:
I thoguht you mentioned that the key is auto populated and has some text to it?

Reply:
Yup, but i went to look it up again, and realise that its using div, and this code should be added by boon. because he did changed it to ul and li. Also, I checked, it seems that the props content is a react element so it wont have a key.